src/Rules/CountryCode.php

```diff
@@ -40,7 +40,7 @@ public function passes($attribute, $value): bool
 
     public function message(): string
     {
-        return __('validation.country_code', [
+        return __('validationRules::messages.country_code', [
```

Comment:
Let's keeping use the `validation` file here and for the other rules.

Reply:
Currently the file is called `validationRules` so should it be
```
'validationRules::validationRules.country_code'
```
or 
```
'validationRules::validation.country_code'
```

Thanks

Reply:
@freekmurze sorry, could you let me know which one you prefer?
Thanks